### PR TITLE
merge from 0.5 to 0.6 [skip ci]

### PR DESCRIPTION
**Pull request by pxLi**
fix https://github.com/NVIDIA/spark-rapids/pull/2193

**NOTE:** this should be merged as a merge commit

### CODE REVIEW

File: docs/download.md [hunk 1]
```diff
@@ -4,15 +4,64 @@ title: Download
...
+	Apache Hadoop 2.10+ or 3.1.1+ (3.1.1 for nvidia-docker version 2)
+	
+	Python 3.6+, Scala 2.12, Java 8 
+
+### Download v0.5.0
+* Download [RAPIDS Spark Package](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/0.5.0/rapids-4-spark_2.12-0.5.0.jar)
+* Download RAPIDS cuDF 0.19.0 for your system: 
+  * [cuDF 11.0 Package](https://repo1.maven.org/maven2/ai/rapids/cudf/0.18.2/cudf-0.19.0-cuda11.jar)
+  * [cuDF 10.2 Package](https://repo1.maven.org/maven2/ai/rapids/cudf/0.18.2/cudf-0.19.0-cuda10-2.jar)
+  * [cuDF 10.1 Package](https://repo1.maven.org/maven2/ai/rapids/cudf/0.18.2/cudf-0.19.0-cuda10-1.jar)
```
Comment (pxLi) [inline docs/download.md:48]: this should be fixed in another PR target to 0.5

**Comment by pxLi** [comment]
build

**Review by GaryShen2008 — APPROVED** [review]

File: docs/download.md (hunk 1 above)
Comment (pxLi) [reply docs/download.md:48]: https://github.com/NVIDIA/spark-rapids/pull/2195
